src/test/scala/io/iohk/ethereum/consensus/ConsensusSpec.scala

```diff
@@ -197,61 +197,6 @@ class ConsensusSpec extends AnyFlatSpec with Matchers with ScalaFutures {
     blockQueue.isQueued(oldBlock3.header.hash) shouldBe true
   }
 
-  it should "get best stored block after reorganisation of the longer chain to a shorter one if desync state happened between cache and db" in new EphemBlockchain {
```

Comment:
This test was removed because testing synchonization between cache and DB does not make sense anymore